Frameworks/VMFleet/VMFleet.psm1

```diff
@@ -8665,4 +9136,152 @@ function SetCacheBehavior
             Remove-PSSession -Session $s
         }
     }
+}
+
+function Get-Salt {
+    $salt = ([char]'a'..[char]'z' + [char]'A'..[char]'Z' + [char]'0'..[char]'9' | Get-Random -Count 4 |% { [char]$_ }) -join ''
+    Write-Host "Generated Salt: $salt"
+    return $salt;
+}
+
+function Get-ArcConfig {
+    [CmdletBinding()]
+    param(
+        [Parameter()]
+        [string]
+        $Cluster = "."
+    )
+
+    $controlPath = Get-FleetPath -PathType Control $Cluster
+    $f = Join-Path $controlPath "arc.json"
+    if (-not (Test-Path $f)) {
+        Write-Host "arc.json does not exist. Current Arc mode: Disabled."
```

Comment:
can we put this under verbose? value understood while doing initial development.